diff --git a/node_modules/@rocicorp/zero/out/zero-cache/src/server/syncer.js b/node_modules/@rocicorp/zero/out/zero-cache/src/server/syncer.js
index 373649e..f0c9786 100644
--- a/node_modules/@rocicorp/zero/out/zero-cache/src/server/syncer.js
+++ b/node_modules/@rocicorp/zero/out/zero-cache/src/server/syncer.js
@@ -1,7 +1,8 @@
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
 import { Resource } from '@opentelemetry/resources';
 import { NodeSDK } from '@opentelemetry/sdk-node';
-import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION, } from '@opentelemetry/semantic-conventions';
+import pkg from '@opentelemetry/semantic-conventions';
+const { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } = pkg;
 import { tmpdir } from 'node:os';
 import path from 'node:path';
 import { pid } from 'node:process';
